src/interfaces/ISablierMerkleBase.sol

```diff
@@ -2,10 +2,10 @@
 pragma solidity >=0.8.22;
 
 import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
-import { IAdminable } from "@sablier/lockup/interfaces/IAdminable.sol";
+import { IAdminable } from "@sablier/lockup/src/interfaces/IAdminable.sol";
 
 /// @title ISablierMerkleBase
-/// @dev This is the base interface for Merkle Lockups and Merkle Instant.
+/// @dev Common interface between Merkle Lockups and Merkle Instant.
```

Comment:
since the repo is now called `airdrop`, should refer from now on to this as: 
```solidity
/// @dev Common interface between Merkle airdrop campaigns.

or

/// @dev Common interface between Merkle airdrop distribution campaigns.

```

Reply:
No. That NatSpec comment is there for developers, not for end users. Developers should be informed about the specific contracts using this common interface.